packages/charts/react-charting/src/utilities/image-export-utils.ts

```diff
@@ -86,95 +96,114 @@ function toSVG(chartContainer: HTMLElement, background: string) {
   };
 }
 
-const LEGEND_RECT_STYLE_PROPERTIES_MAP = {
-  'background-color': 'fill',
-  'border-color': 'stroke',
-};
+const LEGEND_CONTAINER_MARGIN_TOP = 8;
+const LEGEND_CONTAINER_MARGIN_START = 12;
+const LEGEND_PADDING = 8;
+const LEGEND_HEIGHT = 32;
+const LEGEND_SHAPE_SIZE = 12.5;
+const LEGEND_SHAPE_MARGIN_END = 8;
```

Comment:
should these be defined inside legends style file

Reply:
moved the constants to legends style file